projects/tuist/features/step_definitions/shared/tuist.rb

```diff
@@ -1,9 +1,13 @@
 # frozen_string_literal: true
 require "open3"
 Given(/tuist is available/) do
-  system("swift", "build")
-  @tuist = ".build/debug/tuist"
-  @tuistenv = ".build/debug/tuistenv"
+  if ENV["CI"].nil?
```

Comment:
This is because we assume that tuist has been built by a previous job, correct? If that's so, I'd rather see it also documented here as it might not be obvious.

Reply:
Good suggestion, will add comment for both!